core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/models/Price.kt

```diff
@@ -2,27 +2,16 @@ package com.revenuecat.purchases.kmp.models
 
 /**
  * Represents a monetary price.
+ * @property formatted Formatted price of the item, including its currency sign. For example $3.00.
+ * @property amountMicros Price in micro-units, where 1,000,000 micro-units equal one unit of the currency.
+ * For example, if price is "€7.99", price_amount_micros is 7,990,000. This value represents
+ * the localized, rounded price for a particular currency.
+ * @property currencyCode Returns ISO 4217 currency code for price and original price.
+ * For example, if price is specified in British pounds sterling, price_currency_code is "GBP".
+ * If currency code cannot be determined, currency symbol is returned.
  */
-public expect class Price {
-    /**
-     * Formatted price of the item, including its currency sign. For example $3.00.
-     */
-    public val formatted: String
-
-    /**
-     * Price in micro-units, where 1,000,000 micro-units equal one unit of the currency.
-     *
-     * For example, if price is "€7.99", price_amount_micros is 7,990,000. This value represents
-     * the localized, rounded price for a particular currency.
-     */
-    public val amountMicros: Long
-
-    /**
-     * Returns ISO 4217 currency code for price and original price.
-     *
-     * For example, if price is specified in British pounds sterling, price_currency_code is "GBP".
-     *
-     * If currency code cannot be determined, currency symbol is returned.
-     */
-    public val currencyCode: String
-}
+public data class Price(
```

Comment:
makes sense, I had to make it a regular class instead of `data` because `copy` exposes it

```
Private primary constructor is exposed via the generated 'copy()' method of a 'data' class. 
```

Reply:
Ah yes. [Kotlin 2.0.20 solves that](https://kotlinlang.org/docs/whatsnew-eap.html#data-class-copy-function-to-have-the-same-visibility-as-constructor), but it'll be a while before we can use that.